src/components/Budget/Budget.js

```diff
@@ -1,259 +1,27 @@
-          query: (...args) => {
-            /*
-              This code is not DRY as it is copied from makeQueryFunction in stripes-components.
-              This is necessary, as makeQueryFunction only referneces query paramaters as a data source.
-              STRIPES-480 is intended to correct this and allow this query function to be replace with a call
-              to makeQueryFunction.
-              https://issues.folio.org/browse/STRIPES-480
-            */
-            const resourceData = args[2];
-            const sortMap = {
-              Name: 'name',
-              Code: 'code',
-            };
-            const index = resourceData.query.qindex ? resourceData.query.qindex : 'all';
-            const searchableIndex = searchableIndexes.find(idx => idx.value === index);
-
-            let cql = searchableIndex.makeQuery(resourceData.query.query);
-            const filterCql = filters2cql(filterConfig, resourceData.query.filters);
-            if (filterCql) {
-              if (cql) {
-                cql = `(${cql}) and ${filterCql}`;
-              } else {
-                cql = filterCql;
-              }
-            }
-
-            const { sort } = resourceData.query;
-            if (sort) {
-              const sortIndexes = sort.split(',').map((sort1) => {
-                let reverse = false;
-                if (sort1.startsWith('-')) {
-                  // eslint-disable-next-line no-param-reassign
-                  sort1 = sort1.substr(1);
-                  reverse = true;
-                }
-                let sortIndex = sortMap[sort1] || sort1;
-                if (reverse) {
-                  sortIndex = `${sortIndex.replace(' ', '/sort.descending ')}/sort.descending`;
-                }
-                return sortIndex;
-              });
-
-              cql += ` sortby ${sortIndexes.join(' ')}`;
-            }
-            return cql;
-          },
-        },
-        staticFallback: { params: {} },
-      },
-    },
-    queryCustom: {
-      initialValue: {
-        budgetIDQuery: 'query=(id=null)',
-        fiscalyearQuery: 'query=(name=*)',
-        fundQueryID: 'query=(id=null)',
-        fiscalyearQueryID: 'query=(id=null)'
-      }
-    },
-    budgetID: {
-      type: 'okapi',
-      records: 'budgets',
-      path: BUDGETS_API,
-      recordsRequired: 1,
-      params: {
-        query: (...args) => {
-          const data = args[2];
-          const newData = `${data.queryCustom.budgetIDQuery}`;
-          if (newData === 'undefined') return undefined;
-          const cql = `${newData} sortby name`;
-          return cql;
-        },
+import React from 'react';
+import Switch from 'react-router-dom/Switch';
+import Route from 'react-router-dom/Route';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import BudgetView from './BudgetView/BudgetViewContainer';
+
+const Budget = ({ match: { path } }) => (
+  <Switch>
+    <Route
+      path={`${path}:budgetId/view`}
```

Comment:
I'm not sure you need whole path as it's nested route

Reply:
It doesn't work without `${path}`